libs/platform/src/lib/components/thumbnail/thumbnail-details/thumbnail-details.component.html

```diff
@@ -0,0 +1,38 @@
+<fd-dialog [attr.dir]="_data.rtl ? 'rtl' : 'ltr'">
+    <fd-dialog-header>
+        <h1 fd-title>{{_data.selectedMedia.title}}</h1>
+    </fd-dialog-header>
+    <fd-dialog-body>
+        <button class="fd-button fdp-thumbnail-button--left" (click)="previous()" [disabled]="leftButtonDisabled" [style.z-index]="1" data-slide="previous"
+        >
+        <i [class]="_data.rtl ? 'sap-icon--slim-arrow-right' : 'sap-icon--slim-arrow-left'"></i>
```

Comment:
We need `aria-label` on the previous/next buttons to make them accessible. Should also add the i18n variant so that the `aria-label` can be translated.

Reply:
Added aria-label tag.